CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 1.0.0 (2017-02-15)
+
+* Added support for pk7s (need to pass gen=7 option to parser)
+* Renamed to pkparse
+* Added generation 7 data (pokemon, moves, abilities, games and items)
+* idNo is now either a 5 digit tid for gen 6, or 6 digit tid for gen 7
+* Will use semver from now on (breaking changes only on major version releases)
+
```

Comment:
The ribbon changes were reverted. I can't think of any breaking changes, but I'll see if @SnowPhoenix can think of any.